components/Activitylist.js

```diff
@@ -1,21 +1,55 @@
 import { activityList } from "@/dummyList/activityList";
-import { styled } from "styled-components";
+import styled from "styled-components";
 import { StyledHeadline } from "./Dashboard";
+import Form from "./Form";
+import { uid } from "uid";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function Activitylist() {
+  // the defaultValue is connected to the dummyList. This is helpful in the future for the mongoDB connection
+  const [activityCards, setActivityCards] = useLocalStorageState(
+    "activityList",
+    {
+      defaultValue: activityList,
+    }
+  );
+
+  function handleSubmitActivity(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    const inputData = {
+      name: data.activityName,
+      id: uid(),
+    };
+    setActivityCards([...activityCards, inputData]);
+
+    event.target.reset();
+  }
+
   return (
     <>
-      <StyledHeadline>Friends</StyledHeadline>
-      <StyledActivitySection>
-        <StyledHeadlineForSubpages>Aktivitäten</StyledHeadlineForSubpages>
-        {activityList.map((activity) => (
-          <StyledList key={activity.id}>
-            <StyledListItem>
-              <StyledListItemHeadline>{activity.name}</StyledListItemHeadline>
-            </StyledListItem>
-          </StyledList>
-        ))}
-      </StyledActivitySection>
+      <main>
+        <StyledHeadline>Friends</StyledHeadline>
+        <StyledActivitySection>
+          <StyledHeadlineForSubpages>Aktivitäten</StyledHeadlineForSubpages>
+          {activityCards.map((activity) => (
+            <StyledList key={activity.id}>
+              <StyledListItem>
+                <StyledListItemHeadline>{activity.name}</StyledListItemHeadline>
+              </StyledListItem>
+            </StyledList>
```

Comment:
right now you create a ul element for every single li element. But we want all li elements in a single ul element. Please update your code accordingly.

Reply:
thank you ! :), yes thats a bit confusing! I've done that too complicated, i would like to have multiple list items for each card like a "plan" button, a "like" button etc.